Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController.swift

```diff
@@ -38,27 +37,34 @@ open class _ChatMessageComposerSuggestionsViewController<ExtraData: ExtraDataTyp
     /// the bottomAnchorView as soon as we compute the height of the
     /// contentSize of the nested collectionView
     public var bottomAnchorView: UIView?
-
+    
+    /// A closure to observer when an item is selected.
     public var didSelectItemAt: ((Int) -> Void)?
-
+    
+    /// Property to check if the suggestions view controller is currently presented.
     public var isPresented: Bool {
         view.superview != nil
     }
-
-    private var collectionViewHeightObserver: NSKeyValueObservation?
-
-    // MARK: - Subviews
-
+    
+    /// The collection view of the commands.
     open private(set) lazy var collectionView = uiConfig
         .messageComposer
         .suggestionsCollectionView
         .init(layout: uiConfig.messageComposer.suggestionsCollectionViewLayout.init())
         .withoutAutoresizingMaskConstraints
-
+    
+    /// The container view where collectionView is embedded.
     open private(set) lazy var containerView: UIView = UIView().withoutAutoresizingMaskConstraints
 
-    // MARK: - Overrides
-
+    // Height for suggestion cell, this value should never be 0
+    // otherwise it causes loop for height of this controller and as a result this controller height will be 0 as well.
+    // Note: This value can be 1, it's just for purpose of 1 cell being visible.
+    private let defaultRowHeight: CGFloat = 44
+    
+    private var frameObserver: NSKeyValueObservation?
+    
+    private var collectionViewHeightObserver: NSKeyValueObservation?
```

Comment:
Even it's private I would add some little documentation. But I guess it's up to you and it's obvious.

Reply:
I think the documentation above and also inside setupLayout looks enough to me :)

Reply:
Ok 👍🏼